Cargo.toml

```diff
@@ -45,9 +45,11 @@ log = "0.4.14"
 mimalloc = { version = "0.1.27", default-features = false }
 num_cpus = "1.13.0"
 parking_lot = "0.11.2"
+path-absolutize = "3.0.14"
```

Comment:
It is _so_ rust that there's a crate for this and not use `Path.absolute()`

Reply:
There's `is_absolute` on `Path`, but not `make absolute`.  Sigh